bin/generate-offline-files.js

```diff
@@ -22,9 +22,9 @@ function processResults(results) {
     .reduce((current, next) => next.concat(current), [])
     .split("\n")
     .filter(item => item)
-    .map(item => item.replace(/^src\//, "/"))
+    .map(item => item.replace(/^src\//, "./"))
```

Comment:
The Service Worker now lives ones directory lower than the index.html (as do all the JS, CSS, SVG resources). Hence, "load relative to the SW".